config/routes.rb

```diff
@@ -1,14 +1,21 @@
 Rails.application.routes.draw do
   resources :foods, only: [:new, :create, :destroy]
-  resources :recipes, only: [:new, :show, :create, :destroy] do
+  resources :recipes, only: [:new, :show, :index, :create, :destroy] do
     resources :recipes_food, only: [:create, :destroy, :update]
   end
```

Comment:
**[OPTIONAL CHANGE]:** I noticed that you have defined individual routes for each CRUD method in the recipes controller. It may be more efficient to use the `resources :recipes` method to generate all of the CRUD routes automatically. ✅

Reply:
@taldr27 Thank you for your suggestion. But as I would not use the update and edit resources, I see no reason to include their required routes. Hence, I would like to go ahead with it as it is and leave this as an optional change.